zetaclient/chains/bitcoin/signer/signer.go

```diff
@@ -105,6 +109,11 @@ func (signer *Signer) GetERC20CustodyAddress() ethcommon.Address {
 	return ethcommon.Address{}
 }
 
+// GetGatewayAddress returns empty address
+func (signer *Signer) GetGatewayAddress() string {
```

Comment:
FWIW: the TSS address will be represented as the gateway address to consolidate the entrypoints for each chain. 

Might not matter for this PR

Reply:
Sounds good. I'll put a note on both methods.

Reply:
https://github.com/zeta-chain/node/pull/2560/commits/e94fd92e535981bea13dd89a2ad4e13a90a9c147